patches/cypress+3.4.1.patch

```diff
@@ -0,0 +1,179 @@
+diff --git a/node_modules/cypress/lib/exec/spawn.js b/node_modules/cypress/lib/exec/spawn.js
```

Comment:
Is this file still needed?

Reply:
Yes, we need this. This is the patch file that we need to make changes to Cypress in node_modules to deal with the pre-commit error that we had.